### PR TITLE
[perl/ru] Russian translation for perl 5

### PR DESCRIPTION
**Pull request by liruoko**
Besides, I dared to add a few words on Unicode support as well as on pragmas strict and warnings.

### CODE REVIEW

File: ru-ru/perl-ru.html.markdown [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+#### Файлы и ввод-вывод
+
+# Открыть файл на чтение или запись можно с помощью функции "open()".
+
+open(my $in,  "<",  "input.txt")  or die "Can't open input.txt: $!";
+open(my $out, ">",  "output.txt") or die "Can't open output.txt: $!";
+open(my $log, ">>", "my.log")     or die "Can't open my.log: $!";
+
+# Читать из файлового дескриптора можно с помощью оператора "<>".  
+# В скалярном контексте он читает одру строку из файла, в списковом --
```
Comment (Oire) [inline ru-ru/perl-ru.html.markdown]: одну

**Comment by Oire** [comment]
@liruoko, wow! One single typo to correct, and I merge it! :) And I'm terribly sorry for _such_ a delay :(

**Comment by liruoko** [comment]
No problem ^_^ 
Corrected.

**Comment by Oire** [comment]
@liruoko, thanks a lot! Excellent work, I'd be glad to see more of your translations :)

**Pull request merged**